sbc_compassion/tools/import_letter_functions.py

```diff
@@ -405,3 +405,33 @@ def is_zip(name):
     """
     ext = os.path.splitext(name)[1]
     return ext.lower() == '.zip'
+
+
+def find_child(env, code):
```

Comment:
What is this for? Seems that the function is not called?

Reply:
it is used in https://github.com/CompassionCH/compassion-switzerland/pull/972

I thought the function was too long and could extract some parts in this function file.
(The same way it was done with func.analyze_attachment)